pandas/core/internals/blocks.py

```diff
@@ -1354,6 +1354,31 @@ def delete(self, loc) -> None:
             # _cache not yet initialized
             pass
 
+    @cache_readonly
+    def array_values(self) -> ExtensionArray:
+        return self.values
+
+    def get_values(self, dtype: DtypeObj | None = None) -> np.ndarray:
```

Comment:
do we not use Optional anymore?

Reply:
i think a code check was added in the last few days, not sure of the specifics